util/masking.py

```diff
@@ -184,7 +188,13 @@ def generate_ice_ring_resolution_ranges(beam, panel, params):
 @lru_equality_cache(maxsize=3)
 def _get_resolution_masker(beam, panel):
     logger.debug("resolution masker cache miss")
-    return ResolutionMaskGenerator(beam, panel)
+    t0 = time.perf_counter()
+    panel = copy.deepcopy(panel)
+    panel.set_px_mm_strategy(SimplePxMmStrategy())
```

Comment:
Should we leave the option of computing with the native strategy? e.g. for cases where you are making a mask manually so don't mind a 2s delay (for metal rings could make a difference to output too)

Reply:
I've modified the implementation so now this can be controlled by setting the new masking parameter `parallax_correction=False` (the default is `parallax_correction=None`, i.e. use whatever `PxMmStrategy` is set on the panel. This depends on https://github.com/cctbx/dxtbx/pull/319.